### DISCUSSION TITLE
A little help.

### DISCUSSION THREAD

**Discussion by Hargard339** [DCC-EX/EX-T3-WiFi#69] [Q&A] [answered]
Hi.
Have all the parts as you listed. 
Downloaded the code from Git
Playing with VSC at the moment. 
Can compile the code with Success. 
Also uploaded to the FireBeetle with no issues.
The issue I am having is the screen flashes On-Off constantly approx. 1 second. No visual on screen.
Have tested the screen by uploading a LCD test screen. That works NP.
Not sure what you are talking about in the Tutorial "Uploading the Code"
Any advice would help (Novice programmer Experienced Electronics).
One thing I noticed in the EX-T3-WIFI-master/src/main.ccp is "onst uint8_t BATTERY_PIN = A2;" I am assuming this is the battery state. for future development.

Regards Antony.

**Comment by Hargard339** [comment]
A little more info. If I remove the SDF card it alters the speed of the On/Off rate.
 And yes the SD is Formatted Fat32 with the Folder "icons" in the root.
and the default.json  in the root.

**Comment by alex-code** [comment]
Did you upload the file system using VSC too?
There's an internal file system as well as the MicroSD card which contains some files needed for the UI.

https://github.com/DCC-EX/EX-T3-WiFi/blob/master/docs/code.md#uploading-the-code

**Comment by Hargard339** [comment]
Hi Thanks for getting Back to me.
NO.
I looked at it before.. No idea how too. Have googled it but could not find any answers.
Will check it out again... 
Thanks.

**Comment by Hargard339** [comment]
The building and Uploading are OK. pics below.
![Success Building](https://user-images.githubusercontent.com/75605011/222814608-8c87e7a4-3b7b-4568-837c-8ce1eb261ab7.jpg)
![Succes uploading](https://user-images.githubusercontent.com/75605011/222814613-b740c33b-6d19-49e6-bc8a-fa2ec5b25305.jpg)

Thanks.

**Reply by alex-code** [reply]
There's two different bits that need uploading.
One is the compiled code and the other is a file system.

In the task menu you need to do `Upload Filesystem Image`

**Comment by Hargard339** [comment, marked as answer]
Thank You. Working :)
The platformIO Icon was missing. had to uninstall  things.
Thanks again.
Antony.
